### ISSUE TITLE
Granular canonicalization metrics dashboard

### ISSUE THREAD

**Issue by Rjected** [paradigmxyz/reth#5337]
### Describe the feature

We have started to measure canonicalization performance at a more granular level:
https://github.com/paradigmxyz/reth/pull/5266

Many new metrics have been added, these should be added in the grafana dashboard

**Comment by solidoracle** [comment]
I'd like to tackle this

**Comment by Rjected** [comment]
> I will take up this issue

Awesome! Assigned, let us know if you need pointers

**Comment by solidoracle** [comment]
> > I will take up this issue
> 
> Awesome! Assigned, let us know if you need pointers

Thanks! Yes, so should I add in metrics-exporter.json under a new panel for these metrics titled "Canonicalization Metrics", or should I add under "Consensus Spec"?

**Comment by Rjected** [comment]
> > > I will take up this issue
> > 
> > 
> > Awesome! Assigned, let us know if you need pointers
> 
> Thanks! Yes, so should I add in metrics-exporter.json under a new panel for these metrics titled "Canonicalization Metrics", or should I add under "Consensus Spec"?

No, this would go into the `overview.json` file which holds all reth metrics, `metrics-exporter` is just added for convenience since it's a useful third party dashboard. I'd recommend getting prometheus / grafana setup locally with a testnet node (for example holesky, which takes minutes to sync), making changes to the dashboard, then submitting a PR
